portfolio/src/main/webapp/style.css

```diff
@@ -31,18 +21,41 @@ table.allTable{
     font-family: Tahoma, Geneva, sans-serif;
 }
 
-#socials-row{
+td#aboutMe-info{
+    font-size: 125%;
+}
 
+img#avatar{
+    height: 300px;
 }
 
-#aboutMe, #projects{
+/* 
+    External Citation:
+    https://www.webfx.com/blog/web-design/circular-images-css/
+*/
+.circular--square {
```

Comment:
Stay consistent with class name conventions – is there a reason you're using two hyphens here as opposed to one for others?